### PR TITLE
Adding a 2D/3D convected vortex case.

### PR DESCRIPTION
**Pull request by esclapez**

### CODE REVIEW

**Comment by esclapez** [comment]
It also seemed that a working EBGodunov input for the FlowPastCylinder was missing.

File: Exec/eb_run2d/inputs.2d.flow_past_cylinder-x-ebgodunov [hunk 1]
```diff
@@ -16,6 +16,7 @@ ns.visc_tol=1.0e-11
 ns.use_godunov=1 
 ns.do_mom_diff=1
 ns.do_cons_trac=1
+ns.do_cons_trac2=1
```
Comment (cgilet) [inline Exec/eb_run2d/inputs.2d.flow_past_cylinder-x-ebgodunov:19]: You have to have ns.do_trac2 =1 for do_cons_trac2 to be relevant. I'll fix the logic in NSB.

Comment (esclapez) [reply Exec/eb_run2d/inputs.2d.flow_past_cylinder-x-ebgodunov:19]: I was finding that weird indeed. It also pops up in PeleLM where we never actually see the IAMR tracers.

File: Exec/run3d/inputs.3d.convectedvortex [hunk 2]
```diff
@@ -0,0 +1,135 @@
+
+#*******************************************************************************
+# INPUTS.2D.TAYLORGREEN
```
Comment (cgilet) [inline Exec/run3d/inputs.3d.convectedvortex:3]: Please fix this header and copy over comment about the scripts.

**Review by cgilet — APPROVED** [review]

**Pull request merged**